config/nrfconnect/chip-module/Kconfig.features

```diff
@@ -130,8 +128,6 @@ endif
 
 config CHIP_DFU_OVER_BT_SMP
 	bool "Enable DFU over Bluetooth LE SMP feature set"
-	imply CHIP_QSPI_NOR if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF52840DK_NRF52840
```

Comment:
The same question, who is going to take care of enabling SPI_NOR/QSPI_NOR? Our documentation says it's enough to enable this config to support DFU, but now it is not going to be true.